src/modules/launcher/Wox.Core/Plugin/PluginInstaller.cs

```diff
@@ -119,8 +117,9 @@ private static PluginMetadata GetMetadataFromJson(string pluginDirectory)
                 {
```

Comment:
Not related to this PR, but as a general comment I've been removing this `#if DEBUG throw new Exception(error)` pattern as I come across it. 
1.  Philosophically I dislike having it behave different in release then debug as this can potentially hide bugs.
2.  I prefer to debug with `Break when thrown` on for exceptions so I will be alerted when exceptions are thrown anyway. 
3.  Throwing a `new` exception hides the stack trace.  This should at the very least be `throw;` or wrapped in `throw new Exception(error, e)` where the original exception is passed into the inner exception.

Reply:
Lets create a new chore.  I've encountered a few of these in the code so far.